omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -511,7 +511,7 @@
 
                 if (encounterDate &gt;= returnDate) {
                 getField('returndate.error').html('The next appointment date must be after the date of this visit ' +
-					getValue('encounterDate.value')).show();
+					encounterDate.toDateString()).show();
```

Comment:
@paleu256 Just use the changeFieldDateToJavascriptDate(getValue('encounterDate.value'))